docs/tutorials/aws-sd.md

```diff
@@ -225,7 +225,6 @@ metadata:
 spec:
     ...
 ```
-
```

Comment:
can you clean this up to have no change here please

Reply:
Sure.